src/screens/points/claim-flow/ClaimRewardsPanel.tsx

```diff
@@ -262,7 +262,10 @@ const ClaimingRewards = ({
       } satisfies RapSwapActionParameters<'claimBridge'>;
 
       const provider = getProviderForNetwork(Network.optimism);
-      const wallet = await loadWallet(address, false, provider);
+      const wallet = await loadWallet({
+        address,
+        provider,
```

Comment:
given the defaults going back to `true`, you will need to explicitly provide `showErrorIfNotLoaded` as `false` here